scripts/backup.pl

```diff
@@ -30,6 +30,7 @@
 use Cwd qw(getcwd abs_path);
 
 # CPAN modules
+use Path::Class;
```

Comment:
I suggest to explicitly import `dir` using `use Path::Class qw(dir)` . So that the next developper won't have to search where `dir()` is coming from.